pep-0001.txt

```diff
@@ -603,11 +603,9 @@ The Type header specifies the type of PEP: Standards Track,
 Informational, or Process.
 
 The format of a PEP is specified with a Content-Type header.
```

Comment:
Explicitly only support `text/x-rst`

```suggestion
The format of a PEP is specified with a Content-Type header.
The only valid value is ``text/x-rst``.
```

Reply:
I ended up using the "must" verbiage suggested by @warsaw